.github/CODEOWNERS

```diff
@@ -1,5 +1,16 @@
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence.
-*       @jmank88 @krebernisak
+# CODEOWNERS Best Practices
+# 1. Per Github docs: "Order is important; the last matching pattern takes the most precedence."
+# Please define less specific codeowner paths before more specific codeowner paths in order for the more specific rule to have priority
 
-pkg/monitoring/ @topliceanu @pedrovsg
+* @smartcontractkit/foundations
+
+/pkg/monitoring/ @topliceanu @pedrovsg
+
+/pkg/codec @nolag
+/pkg/loop @jmank88 @krehermann
+/pkg/services @jmank88
+/pkg/sqlutil @jmank88
+/pkg/timeutil @jmank88
+
+# Keystone
+/pkg/values/ @cedric-cordenier
```

Comment:
Also pkg/workflows (+@krehermann) and pkg/capabilities 🙏

Reply:
Did I get it right?